docs/README.md

```diff
@@ -9,3 +9,4 @@ Krustlet has a lot of documentation. A high-level overview of how it’s organiz
 - [The introduction](intro/README.md) takes you by the hand through a series of steps to run an application on Krustlet. Start here if you’re new to Krustlet.
 - [Topic guides](topics/README.md) discuss key topics and concepts at a fairly high level and provide useful background information and explanation.
 - [Community Guides](community/README.md) teach you about the development process for Krustlet and how you can contribute to the project.
+- [How To Guides](how-to/README.md) explain how to run Krustlet with various types of Kubernetes clusters.
```

Comment:
I can imagine us writing other HOWTOs unrelated to various Kubernetes cluster setups. How to mount volumes into your WASI pod, running with multiple krustlets, etc.

Reply:
Fixed!